setup.py

```diff
@@ -20,6 +20,7 @@
     "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

Comment:
Do we need to note something in the docs or the readme, that 3.10 is only supported when the neo4j 5.x driver is used? Or are we supporting 3.10 for all versions?

Reply:
Good points. We need to add a new badge to the readme, and to the compat matrix in the docs. I think pip will handle the fact that 5.X should only be used with py3.10

Reply:
done